protos/containers/v1/containers.proto

```diff
@@ -45,12 +45,15 @@ message Container {
 	string command = 4;
 	uint64 cpu_time = 5;
 	uint64 memory_usage = 6;
-	uint64 pids_current = 7;
-	uint64 pids_limit = 8;
-	repeated string labels = 9;
-	repeated Port ports = 10;
-	string platform = 11;
-	HostConfig host_config = 12;
+	uint64 memory_limit = 7;
```

Comment:
nit: Maybe we should add a comment to indicate that the outter fields are deprecated